### PR TITLE
Somestyle

### PR DESCRIPTION
**Pull request by JanaDrieschner**

### CODE REVIEW

File: components/Heading/index.js [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+  height: 80px;
+  background-color: #ecf2e4;
+  color: #708a81;
+
+  margin: 0;
 `;
+
 export default Heading;
+
+// color: #8db9aa; color: #708a81;
```
Comment (stefankorder) [inline components/Heading/index.js:21]: Please delete code you don't use anymore.

File: components/LinksHomepage/index.js [hunk 2]
```diff
@@ -5,23 +5,42 @@ import Link from "next/link";
...
+    background-color:#8db9aa;
+    box-shadow: 0px 15 px 20px FaRegIdBadge(13, 240, 252, 0.4);
+    color: #86887b;
+    transform: translateY(-7px);
+  }
+`;
 
-  padding: 12px 8px;
-`;*/
+//#dd8e58;
```
Comment (stefankorder) [inline components/LinksHomepage/index.js:46]: Please delete code you don't use anymore.

File: styles.js [hunk 3]
```diff
@@ -9,5 +9,9 @@ export default createGlobalStyle`
   body {
     margin: 0;
     font-family: system-ui;
+    background-color:#ecf2e4;
+   
   }
 `;
+
+//e3dcd2 #f9f7f0;
```
Comment (stefankorder) [inline styles.js:17]: Please delete code you don't use anymore.

**Review by stefankorder — CHANGES_REQUESTED** [review]
LGTM! Just a few lines of out commented code that I would recommend you to delete.

**Review by mpagels — APPROVED** [review]
Very very, good Jana.
Keep it up!

**Pull request merged**